### DISCUSSION TITLE
Run detekt on build.gradle.kts files (including root and buildSrc)

### DISCUSSION THREAD

**Discussion by eygraber** [detekt/detekt#3902] [Q&A] [answered]
Is this possible? I tried hacking around with `input` but I couldn't get it to work reliably.

**Comment by cortinico** [comment, marked as answer]
Yes it's possible. Definitely for rules without type resolution.
Rules with type resolution instead probably requires more setup and I'm not sure how to easily configure those (and if they're even valuable for a `build.gradle.kts` file).

> but I couldn't get it to work reliably.

Can you provide more context?

**Reply by BraisGabin** [reply]
Some time ago I was doing some refactors aorund the plugin and I can confirm tht `input` is only used with plain detekt. It's ignored completely in the tasks that uses type-resolution.
